src/oatpp/network/tcp/server/ConnectionProvider.cpp

```diff
@@ -43,6 +43,29 @@
   #endif
 #endif
 
+#if _WIN32_WINNT < 0x0600
+  const char * inet_ntop (int af, const void *src, char *dst, socklen_t cnt) {
```

Comment:
Also, could you note where this code comes from? We should at least attribute the original author with a link to its original work.